### PR TITLE
add region exclusion

### PR DESCRIPTION
**Pull request by mcalhoun**
## what

* Add a tag to allow excluding config rules by region

## why
* Some rules such as `INCOMING_SSH_DISABLED` are not available in `an3`

### CODE REVIEW

File: modules/cis-1-2-rules/main.tf [hunk 1]
```diff
@@ -48,3 +51,9 @@ module "aws_config_rules_yaml_config" {
 
   context = module.this.context
 }
+
+data "aws_region" "current" {}
+module "utils" {
```
Comment (aknysh) [inline modules/cis-1-2-rules/main.tf:56]: add an empty line

Comment (mcalhoun) [reply modules/cis-1-2-rules/main.tf:57]: e504453ebf979bf331dfbac1f41464eea42120fa

**Review by aknysh — APPROVED** [review]

**Pull request merged**